app/models/project.rb

```diff
@@ -491,6 +491,8 @@ def check_status(removed = false)
       update_attribute(:status, 'Removed')
     elsif platform.downcase != 'packagist' && [400, 404].include?(response.response_code)
       update_attribute(:status, 'Removed')
+    elsif platform.downcase == "conda" && response.response_code == 404
```

Comment:
I don't think you need this. The if check above it covers this.

Reply:
Which one specifically? those two are for packagist..

Reply:
`elsif platform.downcase != 'packagist' && [400, 404].include?(response.response_code)` covers it

Reply:
OH it's `!=` Damn

Reply:
For those following along: I enabled ligatures in my code editor, so `!=` looked like a wide `≠`, but my brain didn't read the `=\= as a  `!=` because it was so wide.

My bad!

Reply:
![](https://tyrelsouza.com/screenshots/bd63350a1ba855814748683295abd59f.png)

![](https://tyrelsouza.com/screenshots/25786d911539b66bc876abcea64b2e16.png)